### ISSUE TITLE
TravisCI: The name 'ReSharperReports' does not exist in the current context

### ISSUE THREAD

**Issue by Alessar** [AppMetrics/Reporting#58]
Error occurred when compiling build script:
`/home/travis/build/AppMetrics/Reporting/build.cake(239,2)`:
error CS0103: The name `ReSharperReports` does not exist in the current context

**Comment by alhardy** [comment]
Yeah that will have been excluded on non-windows builds. I'm waiting for VSTS to go live with public builds before cleanup this up, otherwise as mentioned potentially using appveyor for non-windows builds as well as windows
